src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +136,27 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testAesdefaultmodepadding() throws InvalidParametersException {
+        String msg = "Test AES Default";
+        String decrptedMsg;
+        byte[] decrypted;
+        decrptedMsg = StringHelper.encrypt(msg);
+
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.DECRYPT_MODE, sks);
+            decrypted = cipher.doFinal(StringHelper.hexStringToByteArray(decrptedMsg));
```

Comment:
for `private static byte[] encryptWithoutSpecifyingAlgorithmParams(String value)`, why not copy the code from StringHelper's encrypt method? It returns a string, not a byte array, which makes things easier.

You also shouldn't need a `encryptWithSpecifyingAlgorithmParams,()` method, since your change to StringHelper's encrypt method already returns a String.